drivers/rotator/WandererRotatorLite.cpp

```diff
@@ -0,0 +1,442 @@
+        int nbytes_read2 = 0;
+        char res1[16] = {0};char res2[16] = {0};
+
+        if ((rc = tty_read_section(PortFD, res1, 'A' ,5, &nbytes_read1)) != TTY_OK)
+        {
+            char errorMessage[MAXRBUF];
+            tty_error_msg(rc, errorMessage, MAXRBUF);
+            LOGF_ERROR("Relative position read error: %s", errorMessage);
+            return;
+        }
+        if ((rc = tty_read_section(PortFD, res2, 'A' ,5, &nbytes_read2)) != TTY_OK)
+        {
+            char errorMessage[MAXRBUF];
+            tty_error_msg(rc, errorMessage, MAXRBUF);
+            LOGF_ERROR("Mechanical position read error: %s", errorMessage);
+            return;
+        }
+        GotoRotatorN[0].value=positiontemp+reversecoefficient*atof(res1)+backlashcompensation;
+
+        GotoRotatorNP.s = IPS_OK;
+        HomeRotatorSP.s=IPS_OK;
+        IDSetNumber(&GotoRotatorNP, nullptr);
+        IDSetSwitch(&HomeRotatorSP, nullptr);
+    }
+    SetTimer(getCurrentPollingPeriod());
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////
+/// \brief WandererRotatorLite::Move
+/// \param cmd
+/// \return
+///
+
+bool WandererRotatorLite::Move(const char *cmd)
+{
+    positiontemp=GotoRotatorN[0].value;
+    int nbytes_written = 0, rc = -1;
+    LOGF_DEBUG("CMD <%s>", cmd);
+    positioncount = abs(atoi(cmd)/1155);
+        if ((rc = tty_write_string(PortFD, cmd, &nbytes_written)) != TTY_OK)
+        {
+            char errorMessage[MAXRBUF];
+            tty_error_msg(rc, errorMessage, MAXRBUF);
+            LOGF_ERROR("Serial write error: %s", errorMessage);
+            return false;
+        }
+        usleep(1200000);
+    SetTimer(150);
+        return true;
+}
+
+
+///////////////////////////////////////////////////////////////////////////////////////////
+/// \brief WandererRotatorLite::sendCommand
+/// \param cmd
+/// \return
+///
+
+bool WandererRotatorLite::sendCommand(const char *cmd)
+{
+    int nbytes_read = 0, nbytes_written = 0, rc = -1;
+    char res[32] = {0};
+    LOGF_DEBUG("CMD <%s>", cmd);
+
+        tcflush(PortFD, TCIOFLUSH);
+        if ((rc = tty_write_string(PortFD, cmd, &nbytes_written)) != TTY_OK)
+        {
+            char errorMessage[MAXRBUF];
+            tty_error_msg(rc, errorMessage, MAXRBUF);
+            LOGF_ERROR("Serial write error: %s", errorMessage);
+            return false;
+        }
+
+
+        if ((rc = tty_read_section(PortFD, res, 'A' ,60, &nbytes_read)) != TTY_OK)
+        {
+            char errorMessage[MAXRBUF];
+            tty_error_msg(rc, errorMessage, MAXRBUF);
+            LOGF_ERROR("Serial read error: %s", errorMessage);
+            return false;
+        }
+        tcflush(PortFD, TCIOFLUSH);
+    res[nbytes_read - 1] = '\0';
+    LOGF_DEBUG("RES <%s>", res);
+    return true;
+}
+
+
+
+bool WandererRotatorLite::saveConfigItems(FILE * fp)
+{
+    INDI::Rotator::saveConfigItems(fp);
+
+    return true;
+}
```

Comment:
No need to override if you don't have any custom properties to save.

Reply:
Fixed,thanks :)